src/tunnel.cc

```diff
@@ -263,9 +270,8 @@ class TunnelStateData: public PeerSelectionInitiator
 
     /// \returns whether the request should be retried (nil) or the description why it should not
     const char *checkRetry();
-    /// whether the successfully selected path destination or the established
-    /// server connection is still in use
-    bool usingDestination() const;
+
+    bool transporting() const;
```

Comment:
Unexplained method with an odd name.

If this is done to bring the code better in sync with FwdState API it needs a reference to those docs:
```suggestion
    /// \see FwdState::transporting()
    bool transporting() const;
```

NP: long-term maybe we should start some sections of ``` /* Foo API (mimic) */``` for AsyncJob/FwdState where these things can go without need to document.

Reply:
Sigh. You are right of course.